ui/analyse/src/wiki.ts

```diff
@@ -5,15 +5,33 @@ export type WikiTheory = (nodes: Tree.Node[]) => void;
 export default function wikiTheory(): WikiTheory {
   const cache = new Map<string, string>();
   const show = (html: string) => {
-    $('.analyse__wiki').html(html).toggleClass('empty', !html);
+    $('.analyse__wiki').toggleClass('empty', !html);
+    $('.analyse__wiki-text').html(html);
     site.pubsub.emit('chat.resize');
   };
 
+  $(window).on('load', () => {
+    const wikiBookField = $('#wikibook-field').first();
+    const wikiBookStorageKey = 'analyse.wikibooks.display';
+    const toggleWikiBookStorage = () => site.storage.boolean(wikiBookStorageKey).toggle();
```

Comment:
Actually, I wanted to do this in the best possible way using Scala features like hook, or typescript snabbdom library, but couldn't come across a better solution. Some assistance here provided kindly might be really helpful. Regards.

Reply:
This file uses plain javascript DOM manipulation, so it makes sense to continue in that direction, as you did.

You don't need `$(window).on('load'`.

Reply:
Actually, I tried it without the `load` event earlier as well and again, but couldn't understand, the event doesn't trigger without it.
Even though the element already exists, any idea, why it could be so, else I may invest some more time to analyze this. Regards.